src/app/app.component.ts

```diff
@@ -87,6 +88,9 @@ export class HeaderComponent<D> implements OnInit {
       this._datepickerContent.datepicker.select(convertedDate);
 
       this._calendar.updateTodaysDate();
+
+      this._datepickerIntl.changes.next();
```

Comment:
@mmalerba This does not have the desired effect; the labels of the weekdays are not updated. Could you point out to me what needs to be changed in the parent the receives this event?

I guess this would need to happen in `calendar.ts`:

```ts
this._intlChanges = _intl.changes.subscribe(() => {
      _changeDetectorRef.markForCheck();
      this.stateChanges.next();
    });
```

https://github.com/angular/components/blob/2a04ccfd46c154cc5a23993cda8dff3474f2248c/src/material/datepicker/calendar.ts#L304-L308

Reply:
It looks like the issue is in month-view: https://github.com/angular/components/blob/master/src/material/datepicker/month-view.ts#L155

It needs to do something similar to what the calendar is doing there, to update this list of weekday names

Reply:
> It needs to do something similar to what the calendar is doing there, to update this list of weekday names

So it should subscribe to `MatDatepickerIntl.changes` too? 

I wonder why the names of the months are updated.